code/datums/callback_invoke.dm

```diff
@@ -0,0 +1,15 @@
+	//This is to reduce copypasta in callback.dm without proc overhead

```

Comment:
You can fit an entire DM game in one macro, Lummox and others have done it before for dev challenges before.

The only thing is you have to write them C++ style, {} and ; which each line ending in a \ to keep the macro moving to the next line